nilearn/reporting/data/js/carousel.js

```diff
@@ -1,25 +1,25 @@
 /**
- * A class representing a map carousel that cycles through different maps.
+ * A class representing a carousel that cycles through different objects.
```

Comment:
do some renaming of variables as this carousel code works with more than just map images